patches/net/minecraft/world/level/block/CauldronBlock.java.patch

```diff
@@ -0,0 +1,10 @@
+--- a/net/minecraft/world/level/block/CauldronBlock.java
++++ b/net/minecraft/world/level/block/CauldronBlock.java
+@@ -53,6 +_,7 @@
+ 
+    @Override
+    protected void receiveStalactiteDrip(BlockState p_152940_, Level p_152941_, BlockPos p_152942_, Fluid p_152943_) {
++      if (!net.neoforged.neoforge.common.CommonHooks.handleDripInfo(p_152943_, p_152941_, p_152942_))
```

Comment:
This `if` statement usage without brackets is too dangerous & confusing in my opinion. Why not refactor it into
```suggestion
+      if (net.neoforged.neoforge.common.CommonHooks.handleDripInfo(p_152943_, p_152941_, p_152942_))
+         return;
```

which is still one hunk but far less likely to get misread or misplaced in an update.